docs/modules/ROOT/pages/cdcExample.adoc

```diff
@@ -0,0 +1,138 @@
+= Change Data Capture with DataStax Enterprise
+
+Capture schema changes in your C* tables and pass them to Apache Pulsar(R) with DataStax Change Data Capture (CDC). This doc will guide you through installing, configuring, and using CDC with C* or DSE using these components:
+
+* C* or DSE environment 
+** https://downloads.datastax.com/#enterprise[DSE 6.8.21]
+** https://cassandra.apache.org/_/download.html[OSS C*]
+* CDC Agent
+** https://github.com/datastax/cdc-apache-cassandra/releases/download/v1.0.5/agent-dse4-pulsar-1.0.5-all.jar[DSE]
+** https://github.com/datastax/cdc-apache-cassandra/releases/download/v1.0.5/agent-c4-pulsar-1.0.5-all.jar[OSS C*]
+* Pulsar
+** https://github.com/datastax/cdc-apache-cassandra/releases/download/v1.0.5/agent-dse4-pulsar-1.0.5-all.jar[DataStax Luna Streaming 2.8.3_1.0.7 Core]
+* Pulsar C* source connector (CSC)
+** https://github.com/datastax/cdc-apache-cassandra/releases/download/v1.0.5/pulsar-cassandra-source-1.0.5.nar[pulsar-cassandra-source-1.0.5.nar]
+
+== Installing and configuring
+
+. Download the https://pulsar.apache.org/download/[Pulsar tarball] and set up a Pulsar cluster. This example uses Pulsar https://pulsar.apache.org/docs/en/standalone/[standalone mode], but you can also use our helpful https://github.com/datastax/pulsar-ansible[Ansible Scripts^].
++
+[source,bash]
+----
+cd $PULSAR_HOME
+bin/pulsar standalone
+----
+
+. Install C*/DSE with your preferred https://docs.datastax.com/en/install/6.8/install/installWhichOne.html[installation method^]. 
+
+. After installing C*/DSE, but before starting the C*/DSE service, set the `Cassandra-env.sh` configuration:
```

Comment:
Emphasize that this is only applicable for CDC agent version 1.03+. Generally speaking, we recommend using the latest CDC agent version (at least version 1.04+) in order to support C* collection data types.